File: C/atv_3.c
#include <stdio.h>
typedef struct empresa{
    char nome;
    int id;
    char cargo;
    float salario;
    char departamento;
    char localizacao;
};

int atualizarInformacoesPessoas(int id, char nome, char cargo, float salario) {
    return 1;
}
int transferirDepartamento(int id, char cargo, char departamento, char localizacao) {
    return 1;
}
int obterInformacoesDepartamento(int id, char cargo, char departamento, char localizacao) {
    return 1;
}

char adicionarEmpregado(char nome, int id, char cargo, float salario, char departamento, char localizacao) {
    return 'a';
}
char removerEmpregado(int id) {
    return 'a';
}
char informacoesEmpregado(int id) {
    return 'a';
}

int main() {
    return 0;
}

File: Java/atv_3.java
public class atv_2 {
	public class empresa {
        private char nome;
        private int id;
        private char cargo;
        private float salario;
        private char departamento;
        private char localizacao;

        public bool atualizarInformacoesPessoas(int id, char nome, char cargo, float salario){
            ...
        }
        public bool transferirDepartamento(int id, char cargo, char departamento, char localizacao){
            ...
        }
        public boolean obterInformacoesDepartamento(int id, char cargo, char departamento, char localizacao){
            ...
        }

        public char adicionarEmpregado(char nome, int id, char cargo, float salario, char departamento, char localizacao){
            ...
        }
        public char removerEmpregado(int id){
            ...
        }
        public char informacoesEmpregado(int id){
            ...
        }
	}
	
    public static void main(String[] args) {
        System.out.println("");
	    }
}  